python/docs/source/getting-started/installation/service/service-operator.md

```
@@ -0,0 +1,451 @@
+Save as `jumpstarter.yaml`, then apply:
+
+```{code-block} console
+$ kubectl apply -f jumpstarter.yaml
+```
+
+## Verify deployment
+
+Check CR status and workloads:
+
+```{code-block} console
+$ kubectl get jumpstarter -n jumpstarter-lab
```

markdownlint-cli2 (0.21.0)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)